.github/ISSUE_TEMPLATE/--feature_request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "Feature Request"
+about: Request/Suggest a new feature in VW
+title: ''
+labels: feature_request
+assignees: ''
+
+---
+
+<!-- Please fill out as much of the template as you can. Delete sections if unneccesary -->
+<!-- 
+Make Sure you've checked out Existing Issues
+-->
+
+## Short Description
+<!-- Describe Your Idea -->
+
+
+## **How This Feature Will Help You/Others**
+
+
+## **Possible Solution/Implementation Details**
+<!-- Describe the solution you'd like. -->
+
+
+## **Example/Links If Any**
+<!-- Link to already similar features if possible -->
+
+## **Can You Help With This Feature**
+<!-- Pull Requests are much Appreciated -->
```

Comment:
I don't think this is necessary in the issue template? We don't want to overload people with Q's

Reply:
I have removed this for now.